compile/create_binaries/Project.toml

```diff
@@ -1,2 +1,3 @@
 [deps]
 PackageCompiler = "9b87118b-4619-50d2-8e1e-99f35a4d4d9d"
+Ribasim = "aac5e3d9-0b8f-4d4f-8241-b1a7a9632635"
```

Comment:
I think you found that the Ribasim dependency was not needed here, could you remove it and also commit the updated manifest after you do?

Reply:
Would be good to also add TOML and LibGit2 deps explicitly (it works since they are stdlibs).

Reply:
Thanks.
I've changed it locally for now.
If the output is valid, I will update this branch.

Reply:
Done